C/plugins/storage/sqlitelb/common/connection.cpp

```diff
@@ -1265,17 +1265,16 @@ int Connection::insert(const std::string& schema,
 			const std::string& table,
 			const std::string& data)
 {
-SQLBuffer	sql;
+string sqlCommand = {};
```

Comment:
Please do not remove this use of SQLBuffer and take the backward step off using raw strings.

Reply:
fixed